pyEpiabm/pyEpiabm/tests/test_household.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+import pyEpiabm as pe
+
+
+class TestHousehold(unittest.TestCase):
+    """
+    Test the 'Household' class.
+    """
+    def test_construct(self):
+        pe.Household((1, 1))
+
+    def test___repr__(self):
+        subject = pe.Household((1, 1))
+        self.assertIsInstance(repr(subject), str)
```

Comment:
Might be good to test whether the actual string is as you expect too, and it's not just printing a null case?